inttest/bootloose-alpine/Dockerfile

```diff
@@ -22,7 +21,9 @@ RUN apk add --no-cache \
   haproxy \
   nginx \
   inotify-tools \
-  ipvsadm
+  ipvsadm \
+  cmd:kubectl
```

Comment:
IMO we could drop the whole kubectl binary. I usually always use `k0s kc` when debugging the inttests locally.